src/main/resources/application.yaml

```diff
@@ -25,7 +25,7 @@ api-schema-file:
   submitanswer-schema: JsonSchema/submitAnswersSchema.json
 api-version-number: 1
 api-config-params:
-  number_of_days_limit: 210
+  number_of_days_limit: 240
```

Comment:
Now, if you put this into environment variable, then I think you would not need to make these PRs. Changing env variable would do the trick :)

Reply:
Thanks @rapolaskaseliscgi will try this in next one